tests/test_consensus.py

```diff
@@ -1,2 +1,37 @@
-class TestBaseConsensusRules(object):
-    pass
+from bigchaindb.consensus import BaseConsensusRules
```

Comment:
I don't know if this was agreed by the team but we have been following the pattern in which we import test dependencies inside the tests instead of at the the top of the file.

http://docs.pylonsproject.org/en/latest/community/testing.html#rule-never-import-the-module-under-test-at-test-module-scope